presto-blackhole/src/test/java/io/prestosql/plugin/blackhole/TestBlackHoleMetadata.java

```diff
@@ -46,7 +46,7 @@
     public void testCreateSchema()
     {
         assertEquals(metadata.listSchemaNames(SESSION), ImmutableList.of("default"));
-        metadata.createSchema(SESSION, "test", ImmutableMap.of());
+        metadata.createSchema(SESSION, "test", ImmutableMap.of(), null);
```

Comment:
I don't like passing nulls. Please pass there some valid variable like: `new PrestoPrincipal(USER, SESSION.getUSER())`

Reply:
Arrghh... I do not like it either. I missed this one.
(I'm quite busy in my role at chief architect right now, but it's not really an excuse.)